tests/php/sync/test_class.jetpack-sync-posts.php

```diff
@@ -134,7 +135,14 @@ public function test_update_post_updates_data() {
 		$remote_post = $this->server_replica_storage->get_post( $this->post->ID );
 		$this->assertEquals( "foo bar", $remote_post->post_content );
 
-		$this->assertDataIsSynced();
+		$posts_sync_module = new Jetpack_Sync_Module_Posts();
+
+		$local_posts = array_map( array(
+			$posts_sync_module,
+			'filter_post_content_and_add_links'
+		), array( get_post( $this->post->ID ) ) );
```

Comment:
`get_post( $this->post->ID )` could be simplified to just `$this->post` - we're already grabbing the full post object in the `setUp` method.

```suggestion
		), array( $this->post ) );
```